src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -347,8 +360,23 @@ private synchronized void regenerateAITasks() {
 				L10N.checkbox("elementgui.common.enable"), 160,
 				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity"));
 
+		restockCondition = new LogicProcedureSelector(this.withEntry("entity/can_restock"), mcreator,
+				L10N.t("elementgui.living_entity.can_restock"), AbstractProcedureSelector.Side.BOTH,
+				L10N.checkbox("elementgui.common.enable"), 250,
+				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity"));
+		rewardXp = new NumberProcedureSelector(this.withEntry("entity/reward_xp"), mcreator,
+				L10N.t("elementgui.living_entity.reward_xp"), AbstractProcedureSelector.Side.BOTH,
+				new JSpinner(new SpinnerNumberModel(4, 0, Integer.MAX_VALUE, 0.5)), 250,
+				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity"));
+		tradingCondition = new ProcedureSelector(this.withEntry("entity/trading_condition"), mcreator,
+				L10N.t("elementgui.living_entity.trading_condition"), AbstractProcedureSelector.Side.BOTH, false,
+				VariableTypeLoader.BuiltInTypes.LOGIC,
+				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity")).setDefaultName(
+				L10N.t("condition.common.true"));
+
 		restrictionBiomes = new BiomeListField(mcreator, true);
 		restrictionBiomes.setValidator(new ItemListFieldSingleTagValidator(restrictionBiomes));
+		professionTrade = new ProfessionListField(mcreator);

```

Comment:
What I meant by adding the exclude button was to add it so users can also use it (meaning adding the true parameter here)

Reply:
It's not possible the way I made it. You don't restrict the professions, you add them. Hence `professionTrade` field, not `restrictProfessionTrade` field

Reply:
It is totally possible to make it work. Currently, you are checking if the list contains the provided(desired) profession. however, to implement the exclude button, you simply need to check if the list doesn't contain the provided profession. From what I checked, it's a simple `!` to add if the exclude button is enabled (and so a simple boolean variable to add inside `LivingEntity.java`)